Subject: xli: use stdargs instead of varargs
From: Robert Schwebel <r.schwebel@pengutronix.de>

Use stdarg.h instead of varargs.h, which is correct for recent
toolchains.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---

Index: rlelib.c
===================================================================
--- rlelib.c.orig
+++ rlelib.c
@@ -12,7 +12,7 @@
  */
 
 #include "xli.h"
-#include <varargs.h>
+#include <stdarg.h>
 #include <ctype.h>
 #include "rle.h"
 
